code/datums/mutations/speech.dm

```diff
@@ -236,7 +237,12 @@
 				value = capitalize(value)
 			message = replacetextEx(message,regex("\b[REGEX_QUOTE(key)]\b","ig"), value)
 		message = trim(message)
-		var/chosen_starting = pick(startings)
-		message = "[chosen_starting] [message]"
+		if(has_startings)
+			var/chosen_starting = pick(startings)
+			message = "[chosen_starting] [message]"
 
 		speech_args[SPEECH_MESSAGE] = message
+
+
+/datum/mutation/human/medieval/no_startings
```

Comment:
this will also be in the random roll pool and it definitely shouldn't (see my other comment on this)